test/configuration.go

```diff
@@ -31,10 +33,41 @@ type Options struct {
 
 // CreateConfiguration create a configuration resource in namespace with the name names.Config
 // that uses the image specified by imagePath.
-func CreateConfiguration(logger *logging.BaseLogger, clients *Clients, names ResourceNames, imagePath string, options *Options) error {
+func CreateConfiguration(logger *logging.BaseLogger, clients *Clients, names ResourceNames, imagePath string, options *Options) (*v1alpha1.Configuration, error) {
```

Comment:
Any reason `CreateLatestService` takes the image as `names.Image` but here it's passed in as `imagePath`?

Reply:
Looking now I actually remember why I did not do this... This function is used by a helper function in e2e tests that takes ImagePath and not Image. To refactor would require touching quite a few e2e tests and make this change very large and likely to conflict multiple times before merged. I will add a TODO here and come back for this in another PR.